broker/config/settings.yml

```diff
@@ -703,6 +703,9 @@ defaults: &defaults
       id: 'bc158c9a-7934-401e-94ab-057082a5073f'
       name: 'v1.0-xsmall'
       free: false
+      async_ops_supporting_parallel_sync_ops: &async_ops_supporting_parallel_sync_ops
```

Comment:
just add one liner with a description in case it is not understood.

Reply:
+1. Too complicated name. Also I feel this should not be a generic name as above. Rather use `supported_features` and specifically call it `bind_during_update`. Also how do you know if the agent is in normal update Vs Major version upgrade? Can agents support bind during both scenarios or should we even distinguish between them? Somehow i am inclined towards leveraging `supported_features` as it fits better.

Reply:
put the comment